source/jormungandr/jormungandr/realtime_schedule/tests/cleverage_test.py

```diff
@@ -0,0 +1,322 @@
+                "departure" : "2016-04-11 14:37:15",
+                "departure_commande" : "2016-04-11 14:40:17",
+                "departure_theorique" : "2016-04-11 14:40:17",
+                "arrival" : "2016-04-11 14:40:17",
+                "arrival_commande" : "2016-04-11 14:40:17",
+                "arrival_theorique" : "2016-04-11 14:40:17",
+                "comment" : "",
+                "realtime" : "1",
+                "waittime" : "00:10:53",
+                "updated_at" : "2016-04-11 14:26:21",
+                "vehicle_id" : "2662",
+                "vehicle_position_updated_at" : "2016-04-11 14:26:21",
+                "origin" : "bdsi"
+            },
+            {
+                "vehicle_lattitude" : "44.814043370749",
+                "vehicle_longitude" : "-0.57294492449656",
+                "waittime_text" : "19 minutes",
+                "trip_id" : "268436310",
+                "schedule_id" : "268468351",
+                "destination_id" : "3341",
+                "destination_name" : "Piscine Chambéry",
+                "departure" : "2016-04-11 14:49:35",
+                "departure_commande" : "2016-04-11 14:49:35",
+                "departure_theorique" : "2016-04-11 14:49:35",
+                "arrival" : "2016-04-11 14:49:35",
+                "arrival_commande" : "2016-04-11 14:49:35",
+                "arrival_theorique" : "2016-04-11 14:49:35",
+                "comment" : "",
+                "realtime" : "1",
+                "waittime" : "00:19:13",
+                "updated_at" : "2016-04-11 14:25:41",
+                "vehicle_id" : "2660",
+                "vehicle_position_updated_at" : "2016-04-11 14:26:21",
+                "origin" : "bdsi"
+            }
+        ]
+        }]
+    return mock_response
+
+class MockRoutePoint(object):
+    def __init__(self, *args, **kwars):
+        self._hardcoded_line_code = kwars['line_code']
+        self._hardcoded_stop_id = kwars['stop_id']
+
+    def fetch_stop_id(self, rt_proxy_id):
+        return self._hardcoded_stop_id
+
+    def fetch_line_code(self):
+        return self._hardcoded_line_code
+
+
+def next_passage_for_route_point_test():
+    """
+    test the whole next_passage_for_route_point
+    mock the http call to return a good response, we should get some next_passages
+    """
+    cleverage = Cleverage(id='tata', timezone='UTC', service_url='http://bob.com/',
+                          service_args={'a': 'bobette', 'b': '12'})
+
+    mock_requests = MockRequests({
+        'http://bob.com/stop_tutu':
+        (mock_good_response(), 200)
+    })
+
+    route_point = MockRoutePoint(line_code='05', stop_id='stop_tutu')
+
+    with mock.patch('requests.get', mock_requests.get):
+        passages = cleverage.next_passage_for_route_point(route_point)
+
+        assert len(passages) == 2
+
+        assert passages[0].datetime == datetime.datetime(2016, 4, 11, 14, 37, 15, tzinfo=pytz.UTC)
```

Comment:
maybe we can add a test quite similar but with a cleverage on the timezone Europe/Paris, the datetime should be also in UTC but with the time shift